static/app/components/events/contexts/utils.tsx

```diff
@@ -382,17 +385,31 @@ export function getContextMeta(event: Event, contextType: string): Record<string
   }
 }
 
+const iconSizesNumber = {
```

Comment:
is there a reason we are redefining iconSizes here instead of using `iconSizes` from `utils/theme.tsx`

Reply:
There's probably a better way to do this, but it's since the ones in theme are suffixed with `px` and the `UserAvatar` component only accepts numbers, but now that I say that aloud I'll just change the `UserAvatar` component